cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -178,8 +178,42 @@ Docker and CDAP Applications
 
 .. include:: ../dev-env.rst  
    :start-line: 7
+   :end-line: 23
 
-.. include:: ../start-stop-cdap.rst  
-   :start-line: 4
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ mvn archetype:generate \
+       -DarchetypeGroupId=co.cask.cdap \
+       -DarchetypeArtifactId=cdap-app-archetype \
+       -DarchetypeVersion=\ |release|
+
```

Comment:
Each of the three formats (Zip, VM, Docker) has a single page dedicated to them describing the steps needed to start developing. I have for Docker (like I did for the VM) replaced what was there with more specific instructions: removed references to Windows in the starting and stoping of CDAP, etc.